source/lib-intro.tex

```diff
@@ -904,6 +904,51 @@
 \pnum
 An implementation may use any technique that provides equivalent observable behavior.
 
+\rSec3[freestanding.entity]{Freestanding entities}
+
+\pnum
+A \defnadj{freestanding}{entity} is a declaration or macro definition
```

Comment:
```suggestion
A \defn{freestanding entity} is a declaration or macro definition
```
Actually, a freestanding entity is not an entity. It is, as it reads, "a declaration or macro definition that [...]". That means that https://github.com/cplusplus/draft/pull/5713#discussion_r939550414 should be applied:
```C++
  namespace views = ranges::views;                                                  // freestanding
```
Even though `views` is a namespace alias, which are not entities, it is a declaration.

Reply:
fixed, and added an index in case folks try to look it up under entity